src/App.jsx

```diff
@@ -1,60 +1,142 @@
+import { useState } from "react"
 import "./App.css"
 
 function App() {
+  const [numberLeft, setNumberLeft] = useState(0);
+  const [numberRight, setNumberRight] = useState(0);
+  const [result, setResult] = useState(0);
+
+  const [operator, setOperator] = useState("");
+
+  const [storeResult, setStoreResult] = useState(0);
+  const [storeLeft, setStoreLeft] = useState(0);
+  const [storeRight, setStoreRight] = useState(0);
+
+
+  const handleNumberClickLeft = (e) => {
+    const clickedNumber = e.target.innerText;
+    if (clickedNumber === "." && numberLeft.toString().includes(".")) {
+      return;
+    }
+    setNumberLeft((prevNumber) => parseFloat(prevNumber + clickedNumber));
+  };
+
+  const handleNumberClickRight = (e) => {
```

Comment:
i'd call this updateRightNumber

Reply:
also, probably better to call this function as `updateRightNumber("3")` rather than with the event